webship-path/src/Webship/Path.hs

```diff
@@ -93,6 +94,6 @@ varParse' e d =
     (Op (pure . e))
     (StateT $ \xs -> case xs of [] -> Left PathConsumed; h : t -> flip (,) t <$> d h)
 
-varInt :: Path Int
-varInt =
-  varParse (T.pack . show) (first T.pack . readEither . T.unpack)
+pathInt :: Path Text -> Path Int
+pathInt =
```

Comment:
Just realised I should make this Show/Read. Not int specific. Doesn't help
with the name though.
On 4 Jun 2016 1:51 PM, "Jacob Stanley" <notifications@github.com> wrote:

> In webship-path/src/Webship/Path.hs
> <https://github.com/charleso/webship/pull/8#discussion_r65796685>:
>
> > @@ -93,6 +94,6 @@ varParse' e d =
> >      (Op (pure . e))
> >      (StateT $ \xs -> case xs of [] -> Left PathConsumed; h : t -> flip (,) t <$> d h)
> >
> > -varInt :: Path Int
> > -varInt =
> > -  varParse (T.pack . show) (first T.pack . readEither . T.unpack)
> > +pathInt :: Path Text -> Path Int
> > +pathInt =
>
> would this clash with anything if it was just int? I think it would look
> nicer
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/charleso/webship/pull/8/files/7503b7f56b474441a47f96bb91281badcc440743#r65796685>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAB5U04JrHefPi3maaMihQ6ZO999LF-vks5qIPZJgaJpZM4IuCyT>
> .
>